src/systemd/cockpit-wsinstance-https@.service.in

```diff
@@ -10,3 +10,13 @@ Slice=system-cockpithttps.slice
 ExecStart=@libexecdir@/cockpit-ws --for-tls-proxy --port=0
 DynamicUser=yes
 Group=cockpit-session-socket
+
+PrivateDevices=true
+ProtectKernelTunables=true
+ProtectControlGroups=true
+MemoryDenyWriteExecute=true
+SystemCallFilter=@system-service
+
+# cockpit-tls does all our outside web related networking, but ws also calls ssh
```

Comment:
Indeed I forgot about this, but.... this is a bit sad when `ConnectTo` isn't enabled by default...

I wonder if we could do something about that.  Isolating `cockpit-ws` to its own network namespace would be a pretty big win.

Reply:
Clearly we need socket-activated beiboot :sweat_smile:

Reply:
Quite honestly, that sounds right. It'd isolate SSH sessions in the same manner as local PAM ones, and would allow us to fully lock down ws. (But I don't want to do that in this PR)